### ISSUE TITLE
Unprefixed `print-color-adjust`

### ISSUE THREAD

**Issue by BenjaminAster** [web-platform-tests/interop#720]
### Description

The CSS [`print-color-adjust`](https://drafts.csswg.org/css-color-adjust-1/#print-color-adjust) property currently needs `-webkit-` prefix in Chromium. I don't see any open issues about this in the Chromium issue tracker; the implementation also seems to exactly match the spec, so I think this can just be unprefixed?

---

This issue is one of seven issues regarding unprefixing CSS properties/values, originally all in #702 (as per https://github.com/web-platform-tests/interop/issues/702#issuecomment-2358173009). The aim is to group the properties together into multiple (but not seven) focus areas.

### Specification

https://drafts.csswg.org/css-color-adjust-1/#print-color-adjust

**Comment by gsnedders** [comment]
https://wpt.fyi/results/css/css-color-adjust?label=master&label=experimental&aligned&q=print-color-adjust

**Comment by mstensho** [comment]
Chromium bug filed: https://issues.chromium.org/issues/376381169

**Comment by emilio** [comment]
Yeah this needs some testing support (since print tests currently assume they print backgrounds).
